claims_db/phclaims/stage/tables/create_stage_mcare_mbsf_ab.yaml

```diff
@@ -0,0 +1,56 @@
+schema: stage
```

Comment:
Just a reminder that you can use just the one YAML file now for creating and loading to SQL. My recommendation is to call it load_... so that it sits right next to the R script that loads the table.

Reply:
right … the YAML is used to create and load. I renamed the file so that it has the same name as the R script, just a different file extension.